packages/ember-simple-auth/tests/unit/services/session-test.js

```diff
@@ -147,19 +135,19 @@ describe('SessionService', () => {
     it("is read from the session's content", function() {
       session.set('some', 'data');
 
-      expect(sessionService.get('data')).to.eql({ some: 'data' });
+      expect(sessionService.get('data')).to.eql({ some: 'data', authenticated: {} });
```

Comment:
This is a valid change IMO since the internal-session initializes it's `content` to `{ authenticated: {} }` while the test session proxy initialized an empty object instead.